src/overlays/actors/ovl_En_Brob/z_en_brob.c

```diff
@@ -103,82 +104,82 @@ void EnBrob_Destroy(Actor* thisx, PlayState* play) {
+void EnBrob_SetupExtrude(EnBrob* this, PlayState* play) {
+    Animation_PlayOnce(&this->skelAnime, &gBrobExtrudeAnim);
     DynaPoly_DisableCollision(play, &play->colCtx.dyna, this->dyna.bgId);
-    this->unk_1AE = 1000;
-    this->actionFunc = func_809CB114;
+    this->yOffset = 1000;
+    this->actionFunc = EnBrob_Extrude;
 }
 
-void func_809CAEA0(EnBrob* this) {
-    Animation_MorphToLoop(&this->skelAnime, &object_brob_Anim_001958, -5.0f);
-    this->unk_1AE = 8000;
+void EnBrob_SetupWobble(EnBrob* this) {
+    Animation_MorphToLoop(&this->skelAnime, &gBrobWobbleAnim, -5.0f);
+    this->yOffset = 8000;
     this->timer = 1200;
-    this->actionFunc = func_809CB218;
+    this->actionFunc = EnBrob_Wobble;
 }
 
-void func_809CAEF4(EnBrob* this) {
-    Animation_MorphToPlayOnce(&this->skelAnime, &object_brob_Anim_000290, -5.0f);
-    this->unk_1AE -= 125.0f;
+void EnBrob_SetupStunned(EnBrob* this) {
+    Animation_MorphToPlayOnce(&this->skelAnime, &gBrobStunnedAnim, -5.0f);
+    this->yOffset -= 125.0f;
     Actor_SetColorFilter(&this->dyna.actor, COLORFILTER_COLORFLAG_BLUE, 255, COLORFILTER_BUFFLAG_OPA, 80);
     Actor_PlaySfx(&this->dyna.actor, NA_SE_EN_GOMA_JR_FREEZE);
-    this->actionFunc = func_809CB2B8;
+    this->actionFunc = EnBrob_Stunned;
 }
 
-void func_809CAF88(EnBrob* this) {
-    Animation_Change(&this->skelAnime, &object_brob_Anim_001750, -1.0f,
-                     Animation_GetLastFrame(&object_brob_Anim_001750), 0.0f, ANIMMODE_ONCE, -5.0f);
-    this->unk_1AE = 8250;
-    this->actionFunc = func_809CB354;
+void EnBrob_SetupRetract(EnBrob* this) {
+    Animation_Change(&this->skelAnime, &gBrobExtrudeAnim, -1.0f, Animation_GetLastFrame(&gBrobExtrudeAnim), 0.0f,
+                     ANIMMODE_ONCE, -5.0f);
+    this->yOffset = 8250;
+    this->actionFunc = EnBrob_Retract;
 }
 
-void func_809CB008(EnBrob* this) {
-    Animation_MorphToLoop(&this->skelAnime, &object_brob_Anim_001678, -5.0f);
+void EnBrob_SetupShock(EnBrob* this) {
+    Animation_MorphToLoop(&this->skelAnime, &gBrobShockAnim, -5.0f);
     this->timer = 10;
-    this->actionFunc = func_809CB458;
+    this->actionFunc = EnBrob_Shock;
 }
 
-void func_809CB054(EnBrob* this, PlayState* play) {
+void EnBrob_Idle(EnBrob* this, PlayState* play) {
     if (this->timer != 0) {
         this->timer--;
     }
     if (this->timer == 0) {
         if (DynaPolyActor_IsPlayerOnTop(&this->dyna)) {
             func_8002F71C(play, &this->dyna.actor, 5.0f, this->dyna.actor.yawTowardsPlayer, 1.0f);
-            func_809CAE44(this, play);
+            EnBrob_SetupExtrude(this, play);
         } else if (this->dyna.actor.xzDistToPlayer < 300.0f) {
-            func_809CAE44(this, play);
+            EnBrob_SetupExtrude(this, play);
         }
     } else if (this->timer >= 81) {
         this->dyna.actor.colorFilterTimer = 80;
     }
 }
 
-void func_809CB114(EnBrob* this, PlayState* play) {
+void EnBrob_Extrude(EnBrob* this, PlayState* play) {
     f32 curFrame;
 
```

Comment:
trying to think if theres a less technical term for this, like "Grow" or "MoveUp". But it is only the model moving up...
Extrude is probably fine, people who are using 3d modeling tools are already very familiar with the term lol, just wondering if there are other ideas

Reply:
I did consider "Grow" and "Shrink" at first until I noticed it really was just a translation of it going up and down. Extrude might not be perfect as it might give off the impression of it being physically extended and not just moved.

I'm open to suggestions. `MoveUp` is fine, though I would probably change `Retract` to `MoveDown` then as well just to stay consistent

Reply:
alright, im fine with move up and move down, but if you want to wait for more opinions thats fair